### PR TITLE
Fix template file to reflect new format for yaml

### PR DESCRIPTION
**Pull request by satran**

### CODE REVIEW

File: riseml/project_template.py [hunk 1]
```diff
@@ -3,9 +3,8 @@
 
 project_init_template = """project: {project_name}
 train:
-  image:
-    name: tensorflow/tensorflow:1.2.0
-    install:
+  image: tensorflow/tensorflow:1.2.0
```
Comment (elmarhaussmann) [inline riseml/project_template.py]: We should use the new syntax with `framework: tensorflow...` if we have a tensorflow image

**Comment by satran** [comment]
@elmarhaussmann updated it.

File: riseml/project_template.py [hunk 2]
```diff
@@ -3,9 +3,8 @@
 
 project_init_template = """project: {project_name}
 train:
-  image:
-    name: tensorflow/tensorflow:1.2.0
-    install:
+  framework: tensorflow
```
Comment (elmarhaussmann) [inline riseml/project_template.py]: best to also add specific tensorflow version:
```
tensorflow:
  version: 1.2.0
```

Comment (satran) [reply riseml/project_template.py]: Done :)

**Pull request merged**